packages/agent-app-sdk/package.json

```diff
@@ -28,8 +28,6 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.2.0",
-    "@livechat/mitt": "0.1.2",
-    "@livechat/postmessage": "^0.3.2",
```

Comment:
Are these libraries not being used?

Reply:
No, they are not used in the `agent-app-sdk`, but they are present in the `widget-sdk-core`. I assume they might have been used in the past, or perhaps the `package.json` file was simply copied from `widget-sdk-core`.